torch/_dynamo/variables/misc.py

```diff
@@ -624,6 +624,64 @@ def __init__(self, fn_cls, **kwargs) -> None:
         super().__init__(**kwargs)
         self.fn_cls = fn_cls
 
+    def as_proxy(self):
+        return self.fn_cls
```

Comment:
Why does this have an as_proxy? We shouldn't be putting autograd.Functions into the graph

Reply:
Yes, this is typo, we don't use it actually, will remove it.